README.md

```diff
@@ -15,31 +15,71 @@ Marsha is a functional, higher-level, English-based programming language that ge
 
 ## Usage
 
-The Marsha compiler can be invoked via a CLI or `pip` module that can also be invoked in Jupyter notebooks.
+The Marsha compiler can be used to compile the syntax using a `pip` module via a terminal or Jupyter Notebook:
 
-<> TODO Demo
-
-<> TODO some instructions
+```bash
```

Comment:
should we also add the make install + marsha <src> syntax?

Reply:
I think we can start with this as it will be easier to consume since presumably they already have python installed